campaignion_wizard/src/WebformStepUnique.php

```diff
@@ -30,4 +30,26 @@ public function validateStep($form, &$form_state) {
     }
   }
 
+  /**
+   * Check whether webforms for a node are captcha protected.
+   *
+   * @param object $node
+   *   The node which’s captcha settings should be checked.
+   *
+   * @return bool
+   *   TRUE if a captcha is set up for this node’s webform otherwise FALSE.
+   */
+  protected function hasCaptcha($node) {
+    if (!module_exists('captcha')) {
+      return FALSE;
+    }
+    $form_id = "webform_client_form_{$node->nid}";
```

Comment:
Is that save? What if there are several forms on the same nid?

Reply:
How can there be more than one webform for a webform node?

Reply:
Well if there can’t, it’s fine ;)